app/presenters/activity_presenter.rb

```diff
@@ -125,9 +129,9 @@ def trackable_name
 
   def trackable_title
     @title ||= if activity.trackable.respond_to?(:title) && activity.trackable.title?
```

Comment:
What's the convention here? I'm thinking this should be in the next line like: https://github.com/dradis/dradis-ce/blob/develop/app/presenters/digest_presenter.rb#L56

Reply:
updated